src/scrapy.cfg

```diff
@@ -0,0 +1,11 @@
+# Automatically created by: scrapy startproject
+#
+# For more information about the [deploy] section see:
+# https://scrapyd.readthedocs.org/en/latest/deploy.html
+
+[settings]
+default = scraper.settings
+
+[deploy]
+#url = http://localhost:6800/
+project = scraper
```

Comment:
These scrapers look special so I'm not going to touch them. What's going on here? If it's generated code, it shouldn't be in the repo, no?